src/devices/UFireIse/UFireIse.cs

```diff
@@ -0,0 +1,391 @@
+        /// </summary>
+        /// <returns>The measured value (reading value) for calibrate the high value</returns>
+        public ElectricPotential GetCalibrateHighReading()
+        {
+            return new ElectricPotential(ReadRegister((byte)Register.ISE_CALIBRATE_READHIGH_REGISTER), UnitsNet.Units.ElectricPotentialUnit.Millivolt);
+
+        }
+
+        /// <summary>
+        /// Dual point uses two measures for low and high points. It needs the measured value (reading value) and the known value (reference value). Calling calibrateProbeLow saves both the reading and reference value.
+        /// When there are high and low calibration points, the device will automatically use them to adjust readings.To disable dual-point adjustment, call reset to remove all calibration data.
+        /// </summary>
+        /// <returns>The measured value (reading value) for calibrate the low value</returns>
+        public ElectricPotential GetCalibrateLowReading()
+        {
+            return new ElectricPotential(ReadRegister((byte)Register.ISE_CALIBRATE_READLOW_REGISTER), UnitsNet.Units.ElectricPotentialUnit.Millivolt);
+        }
+
+        /// <summary>
+        ///  Resets all the stored calibration information
+        /// </summary>
+        public void Reset()
+        {
+            WriteRegister((byte)Register.ISE_CALIBRATE_SINGLE_REGISTER, null);
+            WriteRegister((byte)Register.ISE_CALIBRATE_REFHIGH_REGISTER, null);
+            WriteRegister((byte)Register.ISE_CALIBRATE_REFLOW_REGISTER, null);
+            WriteRegister((byte)Register.ISE_CALIBRATE_READHIGH_REGISTER, null);
+            WriteRegister((byte)Register.ISE_CALIBRATE_READLOW_REGISTER, null);
+        }
+
+        /// <summary>
+        /// Sets all the values, in mV, for dual point calibration and saves them in the devices's EEPROM
+        /// </summary>
+        /// <param name="refLow">the reference low point. Range: -1024 mV to 1024 mV</param>
+        /// <param name="refHigh">the reference high point. Range: -1024 mV to 1024 mV</param>
+        /// <param name="readLow">the measured low point. Range: -1024 mV to 1024 mV</param>
+        /// <param name="readHigh">the measured high point. Range: -1024 mV to 1024 mV</param>
+        public void SetDualPointCalibration(ElectricPotential refLow, ElectricPotential refHigh, ElectricPotential readLow, ElectricPotential readHigh)
+        {
+            if (refLow.Millivolts >= -1024 && refLow.Millivolts <= 1024 &&
+                refHigh.Millivolts >= -1024 && refHigh.Millivolts <= 1024 &&
+                readLow.Millivolts >= -1024 && readLow.Millivolts <= 1024 &&
+                readHigh.Millivolts >= -1024 && readHigh.Millivolts <= 1024)
+            {
+                WriteRegister((byte)Register.ISE_CALIBRATE_REFLOW_REGISTER, Convert.ToSingle(refLow.Millivolts));
```

Comment:
consider making these properties since you got both getter and setter

Reply:
It should not be properties  because the  refLow, refHigh, readLow and readHigh values are only relevant when running the function.